CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- ⚠️  Flatbuffers are now a hard dependency for building and running VAST.
+  The segment store and archive now use a flatbuffers table for their
+  on-disk persistent state.
```

Comment:
```suggestion
- ⚠️ Flatbuffers are now a hard dependency for building and running VAST.
  The segment store and archive now use a flatbuffers table for their
  on-disk persistent state. [#972](https://github.com/tenzir/vast/pull/972)
```

Also, should you rebase the last commit: Please remove the trailing dot from the commit message ;)